microsoft-edge/devtools-guide-chromium/device-mode/index.md

```diff
@@ -214,7 +214,7 @@ You can also set network throttling from the **Performance** panel:
 
  If your page depends on geolocation information from a mobile device to render properly, provide different geolocations using the geolocation-overriding UI.
 
-1. Select **Customize and control DevTools** (`...`) > **More tools** > **Sensors**.
+1. Select **Customize and control DevTools** (**+**) > **More tools** > **Sensors**.
```

Comment:
something's wrong here i believe.  disentangle the two:
* more tools
* customize and control DevTools